mmengine/model/averaged_model.py

```diff
@@ -258,5 +258,6 @@ def avg_func(self, averaged_param: Tensor, source_param: Tensor,
             steps (int): The number of times the parameters have been
                 updated.
         """
-        momentum = max(self.momentum, self.gamma / (self.gamma + self.steps))
+        momentum = max(self.momentum,
+                       self.gamma / (self.gamma + self.steps.item()))
```

Comment:
Is it necessary to call `item()`?

Reply:
I got an error in some conditions but I cannot remember how could it happened :rofl:. But I think it could be more proper to compare the builtin variable with the builtin function `max`